src/common/options/mgr.yaml.in

```diff
@@ -83,60 +83,59 @@ options:
-    service (RGW, rbd-mirror etc)
+  desc: Period in seconds from the last beacon received by the Mmanager after which the
+    state of monitored services (RGW, rbd-mirror etc) are dropped
   default: 1_min
   services:
   - mgr
 - name: mgr_debug_aggressive_pg_num_changes
   type: bool
   level: dev
-  desc: Bypass most throttling and safety checks in pg[p]_num controller
+  desc: Bypass most throttling and safety checks in the Manager's ``pg_num`` and ``pgp_num`` logic
   default: false
   services:
   - mgr
 - name: mgr_max_pg_num_change
   type: int
   level: advanced
-  desc: maximum change in pg_num
+  desc: Maximum ``pg_num`` step
   default: 128
   services:
   - mgr
   with_legacy: true
 - name: mgr_max_pg_creating
   type: uint
   level: advanced
-  desc: bound on max creating pgs when acting to create more pgs
+  desc: Max number of placement groups to create at once
   default: 1024
   services:
   - mgr
 - name: mgr_module_path
   type: str
   level: advanced
-  desc: Filesystem path to manager modules.
-  fmt_desc: Path to load modules from
+  desc: Filesystem path to Manager modules.
   default: @CEPH_INSTALL_DATADIR@/mgr
   services:
   - mgr
 - name: mgr_standby_modules
   type: bool
   default: true
   level: advanced
-  desc: Start modules in standby (redirect) mode when mgr is standby
-  long_desc: By default, the standby modules will answer incoming requests with a
-    HTTP redirect to the active manager, allowing users to point their browser at any
-    mgr node and find their way to an active mgr.  However, this mode is problematic
-    when using a load balancer because (1) the redirect locations are usually private
-    IPs and (2) the load balancer can't identify which mgr is the right one to send
-    traffic to. If a load balancer is being used, set this to false.
+  desc: Start modules in standby (redirect) mode when a Manager is standby
+  long_desc: By default, modules managed by a standby Manager will answer incoming requests with a
+    HTTP redirect to the active Manager.  This allowing users to point their browser at any
+    Manager and find their way to the active Manager.  However, this mode is problematic
+    when using a load balancer because (1) the redirect targets are usually private
+    IP addressess and (2) the load balancer can't identify which Manager to which to send
+    traffic. If a load balancer is being used, set this to false.
 - name: mgr_disabled_modules
   type: str
   level: advanced
   desc: List of manager modules never get loaded
-  long_desc: A comma delimited list of module names. This list is read by manager
-    when it starts. By default, manager loads all modules found in specified 'mgr_module_path',
-    and it starts the enabled ones as instructed. The modules in this list will not
-    be loaded at all.
+  long_desc: A comma delimited list of module names. This list is read by the Manager
```

Comment:
For a future PR (I don't want to waste the time that has already been spent in passing the checks):
s/comma delimited/comma-delimited/

This is not worth doing a force push over.